ydb/core/kqp/common/kqp_tx_manager.h

```diff
@@ -49,6 +51,10 @@ class IKqpTransactionManager {
     virtual EShardState GetState(ui64 shardId) const = 0;
     virtual void SetState(ui64 shardId, EShardState state) = 0;
 
+    virtual NTopic::TTopicOperations& GetTopicOperations() = 0;
```

Comment:
const NTopic::TTopicOperations& GetTopicOperations() const

For changing use SetTopicOperations(...)